### ISSUE TITLE
There is no visual label name for the split button present under "A SplitButton controlling text color in a RichEditBox" heading.

### ISSUE THREAD

**Issue by ayushjai19** [microsoft/WinUI-Gallery#2084]
### Which version of the app?

WinUI 3 Gallery

### Description

Environment Details:
OS Version: Dev (OS Build 27881.1000)
App: WinUI 3 Gallery
App Version: 2.6.0.0 (SDK 1.7)

Repro Steps:
Launch the WinUI 3 Gallery app.
Navigate to search box using tab key.
Type 'SplitButton' in search box and hit enter.
Navigate to 'SplitButton' card and activate it.
Navigate to "A SplitButton controlling text color in a RichEditBox" heading.
Now observe the split button.
Actual Result:
There is no visual label name for the split button present under "A SplitButton controlling text color in a RichEditBox" heading.

Note: Same issue is observed for the split button present under "A SplitButton with text" also.

Expected Result:
There should be a proper label name for the split button present under "A SplitButton controlling text color in a RichEditBox" heading.

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

### Windows version

Windows 11 24H2 (26100)

**Comment by niels9001** [comment]
This is by design, as this shows how `SplitButton` can be used to select a color. The same pattern is shown in apps like Paint.
